tests/base.py

```diff
@@ -324,6 +328,8 @@ def setUp(self, config_dict='config_dict'):
                         {'id': '129', 'key': 'cost', 'defaultValue': '10.99', 'type': 'double'},
                         {'id': '130', 'key': 'count', 'defaultValue': '999', 'type': 'integer'},
                         {'id': '131', 'key': 'variable_without_usage', 'defaultValue': '45', 'type': 'integer'},
+                        {'id': '132', 'key': 'object', 'defaultValue': '{"test": 12}', 'type': 'string',
```

Comment:
Can we add another variable here that has `type` json. That will be to confirm that it is parsed correctly.